### DISCUSSION TITLE
Payment method for artists.

### DISCUSSION THREAD

**Discussion by WiseSmil3** [KRTirtho/spotube#2115] [Ideas]
In the analytics panel or under each artist add a option go connect to the artist support page to buy them a coffee. I want to support all the artists directly.

Roy thanks for this wonderful app. Respect. You always got my support.

**Comment by KRTirtho** [comment]
This is still a work in progress. Because there's no portal/wallet service that have all artists in Spotify. So, it's kind of hard for me to find where people can directly pay them

**Comment by NickDangerous** [comment]
I love this idea and that you are even thinking about implementing it. There are multiple ways to pay it back to artists and it would be nice to have them easily accessible. You could link to places to buy tracks or albums. I like to buy on Bandcamp, when possible. To find this on Amazon or sth would probably be easier. Also shows are a way to earn their money for many musicians.
I think this information could be placed more prominently, than the bottom of the page.  Just next to the download and favoritce buttons.  You could make Spotube to be recogniced as more than an app to use Spotify for free. When I listen to a track for the 100th time or if I dowload the fifth track of some artist, I would want to pay for it and I'd be happy to be remined to do so. Will keep thinking about this.

**Reply by candideu** [reply]
The 100th play reminder is a great idea!

Not everyone is on Bandcamp (and I've found users posing as other artists), so there could be links to iTunes and Qobuz as well.

Spotube could also be linked to the MusicBrainz API, which has fields indicating purchase links for certain tracks and albums.
